docroot/modules/custom/va_gov_facilities/src/EventSubscriber/FacilitiesSubscriber.php

```diff
@@ -62,4 +123,74 @@ protected function clearNormalStatusDetails(EntityInterface $entity): void {
     }
   }
 
+  /**
+   * Archive related content when a facility has been archived.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   Entity.
+   */
+  protected function archiveRelatedFacilityContent(EntityInterface $entity): void {
```

Comment:
```suggestion
  protected function archiveRelatedHealthFacilityContent(EntityInterface $entity): void {
```
If in the future we make this more broad, we can make the name more generic, but for now it is pretty specific.